inlong-dashboard/src/metas/sources/defaults/PostgreSQL.ts

```diff
@@ -151,4 +153,16 @@ export default class PostgreSQLSource
   })
   @I18n('meta.Sources.PostgreSQL.decodingPluginName')
   decodingPluginName: string;
+
+  parse(data) {
+    data.tableNameList = data.tableNameList.join(',');
```

Comment:
Do not destroy the original variable when modifying the data, you can simply copy the data, for example `let obj = {...data}`